internal/install/recipe_installer.go

```diff
@@ -241,6 +241,16 @@ func (i *RecipeInstaller) installRecipes(m *types.DiscoveryManifest, recipes []t
 		"recipe_count": len(recipes),
 	}).Debug("installing recipes")
 
+	isHostTarget := func(recipe types.Recipe) bool {
+		for _, target := range recipe.InstallTargets {
+			if target.Type != types.OpenInstallationTargetTypeTypes.HOST {
+				return false
+			}
+		}
+
+		return true
+	}
+
```

Comment:
I think this looks good. My only thought is I wonder if this should be inverted to look for `isApplicationTarget` specifically. This will need to change in the future once we have more than `HOST` and `APPLICATION`, so this is really just a nit at the present moment.

Reply:
But not necessarily worth holding up this PR over.

Reply:
I think you're right.  I'll plan to make that change in the other issue where we print the output at the bottom of the install flow, since I think its the logic that you mention, and also solves this case here.